tools/updateDerivativesTest.php

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/php
+<?php
+
+require_once __DIR__ . "/../vendor/autoload.php";
+require_once "generic_includes.php";
+require_once "Archive/Tar.php";
+
+/**
+ * Testing edits to derivative database/files
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Sessions
+ * @author     Loris team <info-loris.mni@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://github.com/aces/Loris
+ */
+class TestEditingDerivatives
+{
+
+    /**
+     *  Testing Blake2bHash generator
+     *
+     * @return void
+     */
+    function testHashGenerator()
+    {
+        $db = \NDB_Factory::singleton()->database();
+
+        $f    = file_get_contents(
+            "/data/loris-mri/data/bids_imports/derivatives/"
+            ."loris_annotations/sub-DCC0001/ses-V01/ieeg/"
+            ."sub-DCC0001_ses-V01_task-test_acq-seeg_annotations.tgz"
+        );
+        $hash = sodium_crypto_generichash($f);
+
+        $dbHash = $db->pselectone(
+            'SELECT Blake2bHash
+            FROM physiological_annotation_archive
+            WHERE PhysiologicalFileID=:id',
+            ['id' => '11']
+        );
+
+        if ($hash = $dbHash) {
+            echo "Hash generator works\n";
+        } else {
+            echo "Hash generator does not work\n";
+        }
+    }
+
+
+    /**
+     * Test that derivative file path is created correctly
+     *
+     * @return void
+     */
+    function testFilepathGeneration()
+    {
+        $db =& \Database::singleton();
+
+        //Add derivative folder to filepath
+        $physioFilePath = "bids_imports/__BIDSVersion_1.2.2/sub-DCC0001/"
```

Comment:
Hard coded

Reply:
@christinerogers This is a test script I'm using to test the functionalities I added and will be removed before merging.